.dir-locals.el

```diff
@@ -0,0 +1,7 @@
+(
+ (nil . (
+         ;; (haskell-mode-stylish-haskell-path . "stylish-haskell")
```

Comment:
This is pending a decision on formatters, see https://github.com/kmonad/kmonad/issues/266#issuecomment-846571515